src/Auth/Actions/userActions.js

```diff
@@ -0,0 +1,54 @@
+import axios from 'axios'
+import { sessionService } from 'redux-react-session';
+import { USER_SERVER } from '../../Utils/Proxy'
+import {
+    LOGIN_USER
+} from './types';
+
+
+export const LoginUser = (credentials, history, setFieldError, setSubmitting) => {
+    // Make checks and get some 
+    return () => {
+    const request =  axios.post(`${USER_SERVER}/signin`,
+        credentials,
+        {
+            headers: {
+                "Content-Type": "application/json"
+            }
+        }
+        ).then((response) => {
+            const { data } = response;
+
+             if (data.status === "Failed") {
+                const { message} = data;
```

Comment:
am not a fan of the error handling here. this will lead to cumbersome code if you want to continue like this. imaging you want to check for 10 fields you will do message.includes("email") for all the fields??